src/components/Dashboard/Rewards.tsx

```diff
@@ -136,60 +137,56 @@ export default function Rewards() {
                   </div>
                 }
               />
-              <Descriptions.Item
-                label={t`Your balance`}
-                labelStyle={labelStyle}
-                children={
-                  <div
-                    style={{
-                      display: 'flex',
-                      flexWrap: 'wrap',
-                      justifyContent: 'space-between',
-                      width: '100%',
-                    }}
-                  >
-                    <div>
-                      {ticketsIssued && (
+              {totalBalance?.gt(0) ? (
```

Comment:
i think we still want to show this if balance is 0 yeah? we could just hide it if a wallet isn't connected. at least that's what i was imagining, might have said the wrong thing when we talked about it last

Reply:
Oh yep totally my mistake here! Thanks for the pickup